apps/web/jest.setup.js

```diff
@@ -0,0 +1,10 @@
+jest.doMock('common/hooks/useErrorHandler', () => ({
+  useErrorHandler: () => ({ handleError: jest.fn() }),
+}))
+
+jest.doMock('next/router', () => ({
+  useRouter: () => ({
+    push: jest.fn(),
+    replace: jest.fn(),
+  }),
+}))
```

Comment:
(Discussion)
Should we move file that relate to transforming data or global mock to another folder ? maybe `__mock__`

Reply:
like `__mock__/jest.setup.js` ?

Reply:
Yes